index.html

```diff
@@ -0,0 +1,121 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="utf-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<title>Quiz</title>
+	<link rel="stylesheet" href="quiz.css">
+</head>
+<body>
+<h1>How Blue Jeff Goldblum?</h1>
+		<!-- card number one -->
+	<article class="card">
+		<header class="cardTitle">
+			<h1>Blue</h1>  
+		</header>	
+			<section class="test"> <!-- image section-->
+				<div class="overlay"><div class="blue"></div></div>
```

Comment:
When you have nested elements like this, your code become much more legible if you add in line breaks instead of having it all on one line.